src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java

```diff
@@ -422,57 +245,56 @@ public void getFixedFieldsOfVisualMaterial_return200Status() {
     String url = getURI(FIXED_FIELDS_CODE_GROUPS_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "006")
-      .param("headerTypeCode", "22")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "006").param("headerTypeCode", "22")
+        .param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
 
-
-
-
   @Test
   public void getFixedFieldsByTag008_return200Status() {
 
     String url = getURI(FIXED_FIELDS_CODE_GROUPS_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "008")
-      .param("headerTypeCode", "31")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "008").param("headerTypeCode", "31")
+        .param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
 
```

Comment:
is the @Test annotation omitted intentionally?

Reply:
It is a bibliographic test. @carmentrazza, could you check it, please?

Reply:
It is an oversight, the  @Test annotation must be added and the error code changed from 200 to 500.
It will be done in another PR.

Reply:
Ok, thanks